ui/ui-components/pages/app/[id]/refresh.tsx

```diff
@@ -337,26 +347,28 @@ export default function Page(props) {
                   </Col>
                 </Row>{" "}
                 <Col>
-                  {testResult.success !== null &&
-                  testResult.success !== false &&
-                  testResult.success !== undefined &&
-                  !testResult.error ? (
-                    <Alert variant="success" className="text-break">
-                      <strong>Test Passed. </strong>Sample Value ={" "}
-                      {testResult.message}
-                    </Alert>
-                  ) : ranTest ? (
-                    <Alert variant="warning">
-                      <strong>Test Failed</strong> {testResult.error}
-                    </Alert>
-                  ) : null}
+                  {editing && (
+                    <>
+                      {testResult.success && !testResult.error && ranTest ? (
+                        <Alert variant="success" className="text-break">
+                          <strong>Test Passed. </strong>Sample Value ={" "}
+                          {testResult.message}
+                        </Alert>
+                      ) : ranTest ? (
+                        <Alert variant="warning">
+                          <strong>Test Failed</strong> {testResult.error}
+                        </Alert>
+                      ) : null}
```

Comment:
Probably not necessary, but if this gets any bigger would be good as its own component.

Reply:
Sounds right! This is currently used here and on `/app/[id]/edit` (though the implementation there is slightly different -- the result never disappears there), so if it gets bigger or gets used a third place I agree.